app/src/main/java/chat/rocket/android/app/chatlist/ChatListAdapter.kt

```diff
@@ -22,20 +22,15 @@ class ChatListAdapter(private var dataSet: MutableList<Chat>, private val contex
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         val chat = dataSet[position]
 
-        holder.userAvatar.setImageURI(chat.userAvatarUri)
+        holder.userAvatar.setImageURI(chat.user.avatarUri)
         holder.chatName.text = chat.name
         holder.lastMessage.text = chat.lastMessage
         holder.lastMessageDateTime.text = DateTimeHelper.getDate(chat.lastMessageDateTime, context)
 
         when (chat.type) {
```

Comment:
We should avoid any kind of processing on the Adapter... When we have the SDK in place, and the final models, we will convert from the DB model to a somekind of ViewModel that have all information needed for the adapter... The adapter work should be just binding information to views, not processing informations...

For now, with fake data this is ok, but lets keep that in mind

Reply:
Thanks for advice! When we have the final model we'll do that, for sure! Thanks!